checker.js

```diff
@@ -31,13 +30,9 @@ const colors = require('colors');
 
 const Promise = require('promise');
 
-// const leven = require('leven');
-
 const sizeOf = require('image-size');
```

Comment:
`image-size` is missing in the dependencies list in the README. Such a list will always be outdated. Therefore pleas _remove the list_. Developers will be able to find the depenencies in `package.json`, others will not care.